first week/array,dic,set.playground/Contents.swift

```diff
@@ -0,0 +1,111 @@
+import UIKit
+
+// 생성 방법들
+
+// array 생성
+var a : Array<Int> = [1,2,3,4,5,6,7]
+var b : [Int] = [1,2,3,4,5,6]
+
+//dictionary 생성
+var c : Dictionary<Int,String> = [3:"똥", 2:"바보"]
+var d : [Int : Double] = [1:2.3, 2:3.3]
+
+//set 생성
+var e : Set<Int> = [1,2,3]
+var f : Set = [4,5,6]
```

Comment:
```suggestion
var a: Array<Int> = [1, 2, 3, 4, 5, 6, 7]
var b: [Int] = [1, 2, 3, 4, 5, 6]

//dictionary 생성
var c: Dictionary<Int, String> = [3: "똥", 2: "바보"]
var d: [Int: Double] = [1: 2.3, 2: 3.3]

//set 생성
var e: Set<Int> = [1, 2, 3]
var f: Set = [4, 5, 6]
```

- 보통 세미콜론(:), 쉼표(,)를  쓸 때는 앞은 붙이고 뒤는 띄우는 indent 가 일반적임